pkg/utils/k8sutil.go

```diff
@@ -453,21 +453,33 @@ func CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic,
 		Do().Into(&f)
 	if err != nil {
 		if k8sErrors.IsNotFound(err) {
+			funcLabels := map[string]string{}
+			for _, label := range labels {
+				k, v := getKV(label)
+				funcLabels[k] = v
+			}
+
+			funcAnnotation := map[string]string{
+				"Description": description,
```

Comment:
As far as I can tell, "Description" isn't one of the defined standard annotations.  Why are you storing the description as an annotation, and why is the annotation key not prefixed with something kubeless-specific?